File: 1921.go
package main

import "sort"

func eliminateMaximum(dist []int, speed []int) int {
	n := len(dist)
	est := make([]int, n)
	for i := 0; i < n; i++ {
		est[i] = (dist[i]-1)/speed[i] + 1
	}
	sort.Ints(est)
	for i := 0; i < n; i++ {
		if est[i] <= i {
			return i
		}
	}
	return n
}

File: _Medium/1921/1921.cpp
#include <bits/stdc++.h>
using namespace std;

typedef long long ll;

class Solution {
 public:
  int eliminateMaximum(vector<int>& dist, vector<int>& speed) {
    int n = dist.size();
    vector<int> att(n);
    for (int i = 0; i < n; i++) {
      att[i] = (dist[i]) / speed[i] + 1;
    }
    sort(att.begin(), att.end());
    for (int i = 0; i < n; i++) {
      if (att[i] <= i) {
        return i;
      }
    }
    return n;
  }
};
int main() {

  Solution test;
  return 0;
}